wandb/integration/keras/utils.py

```diff
@@ -0,0 +1,75 @@
+import os
```

Comment:
Add module level docstring

Reply:
@parambharat We don't add module-level docstring for the Keras integration. Is this something specific to this function, or the whole Keras integration module itself?

Reply:
Well, It's generally a good practice to have module level docstrings for all python modules. I'll defer to your call on whether you want to have it for just `utils.py` or for the whole keras integration module.